docs/advanced_usage.rst

```diff
@@ -281,7 +281,7 @@ Where environment variables are not used for configuring the tracer, the instruc
 
 Resolving deprecation warnings
 ------------------------------
-Before upgrading, it’s a good idea to resolve any deprecation warnings raised by your project.
+Before upgrading, it's a good idea to resolve any deprecation warnings raised by your project.
```

Comment:
?

Reply:
I was trying to find random unicode characters floating around the repo because of unicode errors with the docs and found this one.

It didn't fix anything but it doesn't need to be unicode anyway  🤷‍♂

Reply:
Alright, I don't like seeing things unrelated to the PR subject being changed.
You're good for once. ;)